django_object_actions/utils.py

```diff
@@ -127,7 +131,7 @@ def _get_action_urls(self):
         return [
             # change, supports the same pks the admin does
             # https://github.com/django/django/blob/stable/1.10.x/django/contrib/admin/options.py#L555
-            url(
+            re_path(
```

Comment:
not blocking: If we're updating syntax, I wonder if we might as well go to the simpler `path`. A lot of people seem to using really weird primary keys. I'd be willing to accept some broken cases if it means we get an opportunity to better document what kind of primary keys to expect. Are people using non-url safe base64 primary keys with `/`? ¯\\\_(ツ)\_/¯

Reply:
I'm all for it, but this would break support for `Django<2.0`. I didn't want to do this in an out-of-the-blue PR ;-).

If you want, I can move to `path` in a separate PR, but I think this should be merged first.